libafl_qemu/build_linux.rs

```diff
@@ -36,12 +36,31 @@ pub fn build() {
     // Else, we default to `x86_64` - having a default makes CI easier :)
     assert_unique_feature!("arm", "aarch64", "i386", "i86_64");
 
-    let cpu_target = if cfg!(feature = "x86_64") {
-        "x86_64".to_string()
-    } else if cfg!(feature = "arm") {
+    // Make sure that we don't have be set for any architecture other than arm
+    // Sure aarch64 may support BE, but its not in common usage and we don't
+    // need it yet and so haven't tested it
+    assert_unique_feature!("be", "aarch64", "i386", "i86_64");
+
+    // Let things slide when running in clippy
+    #[cfg(not(feature = "clippy"))]
+    #[cfg(not(feature = "usermode"))]
+    #[cfg(feature = "be")]
+    compile_error!("be mode not supported for system-mode");
```

Comment:
Why wouldn't we support be for full system emulation (eventually)?

Reply:
This should be ok now. Effectively, we can just ignore `be` for system-mode as QEMU takes care of it for us.